pkg/chains/signing.go

```diff
@@ -175,6 +180,8 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 				logger.Error(err)
 				continue
 			}
+			logger.Debugf("Recording count of signed messages for %v", obj)
```

Comment:
Trace level?

Reply:
The reason I used Debug level was because, I didn't find trace level used anywhere in Tekton project. On cross checking the logger library it seems like TRACE level logging is not available for the logger used. Let me know if you want to remove these loggers completely, although I think debug level would good enough as it would help with debugging customer issues faster. Done remaining changes suggested.

Reply:
I think we can remove debug logs from here.

Reply:
Removed all debug logging.